google/cloud/storage/bucket.py

```diff
@@ -481,11 +480,11 @@ def bucket(self):
 
     @property
     def public_access_prevention(self):
-        """Setting for public access prevention policy. Options are 'unspecified' (default) or 'enforced'.
-            More information can be found at https://cloud.google.com/storage/docs/public-access-prevention
+        """Setting for public access prevention policy. Options are 'inherited' (default) or 'enforced'.
+            See: https://cloud.google.com/storage/docs/public-access-prevention
```

Comment:
Still needs the blank line to separate summary from body.

Reply:
Ah strange, I think the blank line caught the linter, which is why I changed it back. But perhaps I had just left white space in the line which was the issue.